scripts/postversion.sh

```diff
@@ -1,29 +1,30 @@
 #!/bin/bash
 set -e
 PACKAGE_VERSION=$(node -p -e "require('./package.json').version")
+ARCHIVE_BRANCH="releases"
+SOURCE_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 
 echo 'postversion tasks'
-prettier --write CHANGES.md # after manually hand-editing this might need fixes
-./scripts/copy-documentation-for-new-release.sh $PACKAGE_VERSION
 
-echo 'set new current/next release id in documentation'
-node ./scripts/set-release-id-in-config-yml.cjs
+# npm publish will generate the pkg/sinon.js that we use below
+echo 'publish to npm'
+git push --follow-tags
+npm publish --dry-run
```

Comment:
I split these up onto two lines because under `set -e` it's functionally the same, except for the gotcha where `git push --follow-tags && npm publish --dry-run` won't abort the script if the first command (`git push`) fails.

It seemed likely that we'd want to abort if `git push` failed here.